modules/local/variant_extractor/main.nf

```diff
@@ -0,0 +1,48 @@
+process VARIANT_EXTRACTOR {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/pysam_pip_variant-extractor:23ef51c8f5e0524a':
+        'community.wave.seqera.io/library/pysam_pip_variant-extractor:23ef51c8f5e0524a' }"
+
+    input:
+    tuple val(meta), path(input)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(fasta_fai)
+
+    output:
+    tuple val(meta), path("*.norm.vcf")   , emit: output
+    path "versions.yml"                   , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    normalize_vcf.py \\
+        $input \\
+        ${prefix}.norm.vcf \\
```

Comment:
There is no `bgzip` in this container?

Reply:
I guess I need to add it right, especially handling with big data will get easier with zipped files

Reply:
done :D

Reply:
Indeed, thanks!!